independent-publisher-connectors/JiraSearch/apiProperties.json

```diff
@@ -0,0 +1,45 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "username",
+          "description": "The username for this api",
+          "tooltip": "Provide the username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "password",
+          "description": "The password for this api",
+          "tooltip": "Provide the password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Apply Host URL",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@queryParameters('hostname').atlassian.net/"
+        }
+      }
+    ],
+    "publisher": "Paul Culmsee",
+	"stackOwner" : "Atlassian"

```

Comment:
@paulculmsee Please remove the space to fix indentation for `stackOwner` property.